### DISCUSSION TITLE
Weblate doing strange deletions

### DISCUSSION THREAD

**Discussion by ERYpTION** [Helium314/SCEE#699] [General]
Weblate seem to be doing some strange things with strings for translation: 

https://translate.codeberg.org/translate/scee/app/da/?checksum=513941c3598b84c0

I assume the deletions are not correct?

I have seem a similar issue in another translation project but don't know the explanation.

**Comment by Helium314** [comment]
I have no idea what is going on. This line hasn't changed in more than a year, and the shown changes simply do not exist in this repo.
Maybe I need to sync the translations to SCEE, and the problem will hopefully go away when the strings file is read again.

**Reply by ERYpTION** [reply]
I have reported a similar issue for OsmAnd: https://github.com/osmandapp/OsmAnd/issues/21596

**Reply by Helium314** [reply]
Oh, so probably it's a bug in Weblate
